src/blueapi/cli/cli.py

```diff
@@ -21,54 +19,36 @@ def main(ctx) -> None:
     if ctx.invoked_subcommand is None:
         print("Please invoke subcommand!")
 
+    config_loader = ConfigLoader(ApplicationConfig)
+
+    ctx.ensure_object(dict)
+    ctx.obj["config"] = (
+        config_loader.load_from_yaml(DEFAULT_YAML_PATH)
```

Comment:
So only allowing loading from DEFAULT_YAML_PATH, and assuming that file is changed (by mounting configMap or by editing), rather than passing -c path/to/config.yaml?

I understand your reasoning but will highlight for Callum so you can discuss with him.